SwiftWisdom/Core/StandardLibrary/Array/Array+Utilities.swift

```diff
@@ -54,3 +54,43 @@ extension Array {
         }
     }
 }
+
+extension CollectionType {
+    public var ip_middleValues: [Generator.Element] {
```

Comment:
@Benuuu @brightredchilli see my comment above -- there's a different function for median that sorts the array and then grabs the middle value using this.

You aren't the first people to be confused by it, but not sure what the fix is. I think it's valuable to be able to get the middle element(s) from an array though.